static/src/styles/core/fonts.styl

```diff
@@ -0,0 +1,3 @@
+// This file must be imported before any files that contain CSS output
+// This is because @import declarations must be the first declarations in CSS files
```

Comment:
Putting the `link` directly in our HTML will prevent us from accidentally breaking the fonts